src/components/ProtectedRoute.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { clearFirestoreData, initializeTestApp } from '../lib/testing';
+import ScholarshipForm from '../pages/ScholarshipForm';
+
+window.MutationObserver = require('mutation-observer');
+
+function renderAtRoute(route) {
+  return render(
+    <MemoryRouter initialEntries={[route]}>
+      <Route path="/scholarships/new" component={ScholarshipForm} />
```

Comment:
A few things: 
1. This should be a **ProtectedRoute**
2. The `path` is technically arbitrary but I'd expect it to be `route`?
3. The `component` is also arbitrary but instead of the **ScholarshipForm** (which has its own requirements) I suggest having this method accept a `component` which in the tests can something as simple as `<h1>Protected header</h1>`

The way this is currently written this looks more like what a test might look like for **AddScholarship** (as that's what this page was renamed to).